### ISSUE TITLE
Verify error, but only when compiled for release?

### ISSUE THREAD

**Issue by GaiusCoffey** [denivip/osmf-hls-plugin#89]
Hi,
This one has me stumped, it may well be something I'm doing, but I can't see what! Any pointers are welcome.
1. I am using Charles web proxy to ensure crossdomain.xml always returns a valid and very permissive crossdomain.xml (no domain security issues)
2. When running in debug, I can view .m3u8 content absolutely perfectly as per
   https://devimages.apple.com.edgekey.net/streaming/examples/bipbop_4x3/bipbop_4x3_variant.m3u8
   https://devimages.apple.com.edgekey.net/streaming/examples/bipbop_16x9/bipbop_16x9_variant.m3u8
3. If I compile the exact same content, using the exact same code, and the exact same media, but with a SWF compiled without debugger, I invariably get a Verify error as per the below

VerifyError: Error #1021: At least one branch target was not on a valid instruction in the method.

```
at org.denivip.osmf.net.httpstreaming.hls::HTTPStreamingMP2TSFileHandler/flushFileSegment()
at org.denivip.osmf.net.httpstreaming.hls::HTTPHLSStreamSource/doSomeProcessingAndGetBytes()
at org.denivip.osmf.net.httpstreaming.hls::HTTPHLSStreamSource/getBytes()
at org.denivip.osmf.net.httpstreaming.hls::HTTPHLSStreamMixer/doSomeProcessingAndGetBytes()
at org.denivip.osmf.net.httpstreaming.hls::HTTPHLSStreamMixer/getBytes()
at org.denivip.osmf.net.httpstreaming.hls::HTTPHLSNetStream/onMainTimer()
at flash.utils::Timer/_timerDispatch()
at flash.utils::Timer/tick()
```

G

**Comment by GaiusCoffey** [comment]
Hi,
I've logged a bug ticket here; https://bugbase.adobe.com/index.cfm?event=bug&id=4109701

Please vote for it!
Thanks,
G

**Comment by lucky-zj** [comment]
hi， GaiusCoffey
      Has this problem been solved yet？ The same problem happened to me。

**Comment by GaiusCoffey** [comment]
No, no progress. I have logged a ticket with Adobe as I suspect it is a plugin issue. 
Please vote on that ticket! (See link above.)
G
